integrations/fake-integration/.port/spec.yaml

```diff
@@ -7,3 +7,29 @@ features:
     resources:
       - kind: fake-department
       - kind: fake-person
+configurations:
+  - name: fakePersonsSeedStart
+    required: false
+    type: integer
+    description: End of range for random amount of fake persons per department created
+    default: -1
+  - name: fakePersonsSeedEnd
+    required: false
+    type: integer
+    description: End of range for random amount of fake persons per department created
+    default: -1
+  - name: fakePersonsBatchSize
+    required: false
+    type: integer
+    description: Batch size of requests against fake 3rd party API
+    default: -1
+  - name: fakePersonsBloatFactor
+    required: false
+    type: integer
+    description: Factor of size to bloat entity (by making the 'bio' string field words length times 60 of factor)
```

Comment:
should be actual size and not bloat. ( raw entity size )

Reply:
Done